enhancements/sdk-fbc-run-bundle.md

```diff
@@ -0,0 +1,279 @@
+
+### Goals
+
+- Operator SDK’s `run bundle` command should work with File-Based Catalogs and be able to install an operator bundle using a valid FBC.
+
+- Operator SDK should insert a bundle image into an existing index image that gets passed in as a parameter in the `run bundle` command.
+
+- Operator SDK's `run bundle` command should still be able support SQLite index images along with the FBC images until `opm registry add` and `opm index` and other SQLite-related commands are removed in a future OpenShift release.
+
+- The CLI for `run bundle` command should remain exactly the same i.e. no user-facing changes.
+
+### Non-Goals
+
+- Operator SDK’s `run bundle` and `run bundle-upgrade` commands share a lot of common functionality and this EP does not discuss the impact on the run bundle-upgrade command. A separate EP is required for the `run bundle-upgrade` command.
+
+- Migrating from old catalog images to newer catalog versions with FBC support is not a goal of this EP.
+
+- Another non-goal, is that `run bundle` is **NOT** intended to help users build and manage File-Based indexes. It is an opinionated command to help users test simple bundle-based execution scenarios with ephemeral indexes.
+
+## Proposal
+
+### User Stories
+
+#### Story 1
+
+As an Operator SDK user, I want to be able to install an operator bundle with File-Based Catalog support using OLM without having to provide a catalog.
+
+#### Story 2
+
+As an Operator SDK user, I want to be able to install an operator bundle with File-Based Catalog support using OLM when I want to use an existing File-Based Catalog as the index image.
+
+#### Story 3
+
+As an Operator author, I want to use the Operator SDK’s `run bundle` command that has support for both File-Based Catalog and SQLite index images.
+
+#### Story 4
+
+As an Operator SDK user, I want to be able to use the Operator SDK’s `run bundle` command to install the user provided bundle image with an index image that does not have FBC support (i.e. SQLite based images).
+
+### Implementation Details/Notes/Constraints [optional]
+
+The Operator SDK currently supports running an operator bundle using OLM via the run bundle command, which is suited for the SQLite-based index format. This enhancement will address how the Operator SDK will support the new File-Based Catalog format for operator catalogs.  More Information on FBCs and how operator bundles are created using Operator SDK can be found at the links below:
+
+- [File-Based Catalogs](https://olm.operatorframework.io/docs/reference/file-based-catalogs/)
+- [File-Based Catalog Issue](<https://github.com/k8s-operatorhub/community-operators/discussions/505>)
+- [Operator-Bundles](<https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md>)
```

Comment:
```suggestion
**NOTE:** To know more about the new format ([FBC](https://olm.operatorframework.io/docs/reference/file-based-catalogs/) see the [discussion](https://github.com/k8s-operatorhub/community-operators/discussions/505) raised on the OperatorHub repository. 
```
Could not it be in the bottom of this section? 
Seems that its an extra info and our goal here is to describe the details of how we will begin to use the new format in the command only. WDYT?

Reply:
@camilamacedo86 This is a short intro describing what a declarative config is and pointing to the links for more context, before we talk in detail about how we are actually creating the declarative config as a part of the `run bundle` command. I prefer to leave it here for intro as it describe the "what" before delving into "how", unless there are strong objections.